tailwind.config.cjs

```diff
@@ -3,12 +3,14 @@ module.exports = {
   theme: {
     extend: {
       fontFamily: {
-        rale: ['Raleway', 'sans-serif'],
+        raleway: ['Raleway', 'sans-serif'],
```

Comment:
Ensure to mark this as a breaking change. Many of us already used the previous value **(`font-rale`)**. For compatibility. Add both entries and will be cleaned during the refactor sessions